examples/quotes-api/cmd/quotes-api/main.go

```diff
@@ -0,0 +1,68 @@
+package main
+
+import "github.com/gin-gonic/gin"
+
+func main() {
+	/*
```

Comment:
I assume that you no longer need this comment block!

Reply:
Yeah, I'll be refactoring this in the next PR.